test/fixtures/append.css

```diff
@@ -1,3 +1,4 @@
+@custom-media --a (--a);
```

Comment:
Sorry I don't see how. No import or something in ` test/fixtures/transform-self-reference.css `